src/main/java/mil/dds/anet/AnetApplication.java

```diff
@@ -277,6 +268,37 @@ public void run(AnetConfiguration configuration, Environment environment) {
             metricRegistry));
   }
 
+  private void runAccountDeactivationWorker(final AnetConfiguration configuration,
+      final ScheduledExecutorService scheduler, final AnetObjectEngine engine)
+      throws IllegalArgumentException {
+    // Check whether the application is configured to auto-check for account deactivation
+    if (configuration.getDictionaryEntry("automaticallyInactivateUsers") != null) {
+      // Check for any accounts which are scheduled to be deactivated as they reach the end-of-tour
+      // date.
+      final String intervalDictKey = "automaticallyInactivateUsers.checkIntervalInSecs";
+      final Integer accountDeactivationWarningInterval =
+          (Integer) configuration.getDictionaryEntry(intervalDictKey);
+
+      if (accountDeactivationWarningInterval == null) {
+        final String errorMsg =
+            "Missing or invalid value '" + intervalDictKey + "' in the configuration";
+        logger.error(errorMsg);
+        throw new IllegalArgumentException(errorMsg);
+      }
+
+      final AccountDeactivationWorker deactivationWarningWorker = new AccountDeactivationWorker(
+          configuration, engine.getPersonDao(), accountDeactivationWarningInterval);
+
+      // Run the email deactivation worker at the set interval. In development run it every minute.
```

Comment:
Please don't run stuff every minute (e.g. see commit https://github.com/NCI-Agency/anet/commit/9ec503e3558caa796c1a696549fbc108b2d042a1 ).

Reply:
I changed the code to run the worker once at the startup (while in development mode) just to check whether it is correctly working. However this may change if we decide to implement another solution (see next remark about possible missed email warnings)